theano/tests/breakpoint.py

```diff
@@ -12,8 +12,8 @@ class PdbBreakpoint(Op):
     conditional breakpoint, inside a theano function, based on a symbolic
     scalar condition.
 
-    @type name: String
-    @param name: name of the conditional breakpoint. To be printed when the
+    :type name: String
+    :param name: name of the conditional breakpoint. To be printed when the
```

Comment:
We did agree to have numpydoc as the format for all docstrings.

If that changed, I was not aware.